docs/mkdocs.yml

```diff
@@ -136,8 +136,8 @@ markdown_extensions:
   - pymdownx.mark
   - pymdownx.smartsymbols
   - pymdownx.snippets:
-      auto_append:
-        - includes/abbreviations.md
+      base_path: ['.','../']
+      check_paths: true
```

Comment:
ensure those files exist when we run this

maybe we consider always deploying docs because of this